apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -30,15 +41,26 @@ const RecentCourses = React.createClass({
           showLink={showAllCoursesLink}
           isRtl={isRtl}
         >
+          {!isTeacher && studentTopCourse && (
+            <StudentTopCourse
+              isRtl={isRtl}
+              assignableName={studentTopCourse.assignableName}
+              lessonName={studentTopCourse.lessonName}
+              linkToOverview={studentTopCourse.linkToOverview}
+              linkToLesson={studentTopCourse.linkToLesson}
+            />
+          )}
           {topFourCourses.length > 0 && (
             topFourCourses.map((course, index) =>
+            <div key={index}>
               <CourseCard
-                key={index}
                 name={course.name}
                 description={course.description}
                 link={course.link}
                 isRtl={isRtl}
               />
+              {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
@breville I've been using marginRight (with the nth element having marginRight: 0), and @islemaster is using flexbox. Here I see spacers being used instead of margins. Wondering if you know if there are any best practices on laying out these gridded systems?